operator/src/test/java/org/keycloak/operator/utils/K8sUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.operator.utils;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.logging.Log;
+import org.awaitility.Awaitility;
+import org.keycloak.operator.v2alpha1.crds.Keycloak;
+import org.keycloak.operator.v2alpha1.crds.KeycloakStatusCondition;
+
+/**
+ * @author Vaclav Muzikar <vmuzikar@redhat.com>
+ */
+public final class K8sUtils {
+    public static Keycloak getDefaultKeycloakDeployment(KubernetesClient client) {
+        return (Keycloak) client.load(K8sUtils.class.getResourceAsStream("/example-keycloak.yml")).get().get(0);
+    }
+
+    public static void deployKeycloak(KubernetesClient client, Keycloak kc, boolean waitUntilReady) {
+        client.resources(Keycloak.class).createOrReplace(kc);
```

Comment:
Where is the namespace here ? is this only deploying on the current client namespace ?

Reply:
Here:
https://github.com/keycloak/keycloak/blob/0fcf13406093b9a5a7fb2e252314548d203d8f19/operator/src/test/java/org/keycloak/operator/ClusterOperatorTest.java#L84
It's the default namespace for the client.

Reply:
:+1